### PR TITLE
Prometheus STS

### PR DESCRIPTION
**Pull request by whenhellfreezes**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

Prometheus as a STS

<br/><br/>

2. What **changes to custom.yaml** are required?

None

<br/><br/>

3. Have you documented any additional setup steps or configurations options?

NA

<br/><br/>

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

Ran in minikube. Did a "kc edit sts ..." to change a value and see that the data persisted.
<br/><br/>

### CODE REVIEW

**Comment by chrisbsmith** [comment]
Since the STS declare a PVC itself, you can also remove the prometheus-pvc file: https://github.com/DecipherNow/helm-charts/blob/release-2.1/dashboard/templates/prometheus-pvc.yaml.

File: dashboard/templates/prometheus.yaml [hunk 1]
```diff
@@ -97,6 +97,11 @@ spec:
...
-        persistentVolumeClaim:
-          claimName: prometheus
+  volumeClaimTemplates:
+  - metadata:
+      name: prometheus-pvc
+    spec:
+      accessModes: [ "ReadWriteOnce" ]
+      resources:
+        requests:
+          storage: 40Gi
```
Comment (chrisbsmith) [inline dashboard/templates/prometheus.yaml]: Can you make this configurable?  With the issues in DI2E recently, this would be good to have.

**Comment by whenhellfreezes** [comment]
> Since the STS declare a PVC itself, you can also remove the prometheus-pvc file: https://github.com/DecipherNow/helm-charts/blob/release-2.1/dashboard/templates/prometheus-pvc.yaml.

The prometheus-pvc.yaml is deleted already in this PR

File: dashboard/templates/prometheus.yaml (hunk 1 above)
Comment (whenhellfreezes) [reply dashboard/templates/prometheus.yaml]: Changes added in new commit.

**Comment by chrisbsmith** [comment]
@sofiashalotenko please take a few minutes to run this through a devinabox test when you can.

**Comment by sofiashalotenko** [comment]
I'm a bit confused about the dashboard versioning.  It looks like dashboard is using "appversion" 3.3.1 so what does the "version" 2.1.5 refer to?

**Comment by moosecanswim** [comment]
@sofiashalotenko I just read this and i think its what we're aiming for https://codefresh.io/docs/docs/new-helm/helm-best-practices/#chart-versions-and-appversions

**Comment by sofiashalotenko** [comment]
lgtm

**Pull request merged**